chart/values.yaml

```diff
@@ -103,6 +103,11 @@ rancherImage: rancher/rancher
 # Defaults to .Chart.appVersion
 # rancherImageTag: v2.0.7
 
+# Optionally use rancher image hash instead of image tag.
+# When present, it will be used instead of rancherImageTag.
+# 
+# rancherImageDigest: <sha256 image digest>
```

Comment:
You'll need to create a docs ticket to ensure the docs are updated for this value. At least if we want to document them. 
https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-references/helm-chart-options